t/uni/package.t

```diff
@@ -36,8 +36,6 @@ our $TODO;
         $ㄅĽuṞfⳐ = 5;
     }
     
-    $압Ƈ'd읯ⱪ = 6;        #'
```

Comment:
I think we should preserve this line, this was testing a string using single quote

Reply:
Restored.